scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -1070,15 +1099,11 @@ BEGIN
       )
       SELECT ', ST_Transform(' 
             || t.missing_srid_start || t.attname || t.missing_srid_end
-            || ',4326)::Geometry(' 
-            || t.geomtype 
-            || ',4326) AS '
+            || ',4326)::Geometry(GEOMETRY,4326) AS '
             || const.geomcol
```

Comment:
What would happen if we say "cartodb tables accept any kind of geometry", the user tries to insert a point and it fails? (does not sounds so perfect)